recipes/xdesign/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "xdesign" %}
+{% set version = "v0.5.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tomography/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: b0779b47c93f29d28b20f68355065c0bb7b470fa990898e62c6571078ad266d7
+
+build:
+  noarch: python
+  number: 2
```

This should start from 0 for new versions and only be bumped when changing an already deployed (to anaconda.org) version:
```suggestion
  number: 0
```